modules/pensions/app/sidekiq/pensions/pension_benefit_intake_job.rb

```diff
@@ -59,7 +59,12 @@ def perform(saved_claim_id, user_account_uuid = nil)
 
       upload_document
 
-      @claim.send_confirmation_email if @claim.respond_to?(:send_confirmation_email)
+      begin
+        @claim.send_confirmation_email if @claim.respond_to?(:send_confirmation_email)
+      rescue e
+        @pension_monitor.track_send_confirmation_email_failure(@claim, @intake_service, @user_account_uuid, e)
+      end
```

Comment:
need to add to unit tests to cover the rescue block

Reply:
@wayne-weibel i'll need assistance with this, as I will be on PTO.